openshift/deploy/job/schema-revert.yml

```diff
@@ -42,8 +42,7 @@ objects:
           - /usr/bin/env
           - bash
           - -c
-          - |
-            sqitch revert -y;
+          - sqitch revert -y || exit 0
```

Comment:
do we really want to ignore revert failures?

Reply:
Currently revert will run on the last deploy job's commit, whether it succeeded or not. So yes. This should be fixed in the Makefile